rmgpy/rmg/main.py

```diff
@@ -1348,6 +1356,60 @@ def readMeaningfulLineJava(self, f):
         return line
     
 ################################################################################
+def processToSpeciesNetworks(obj,reactionSystem,coreSpecies):
```

Comment:
Is there a reason these are module level functions rather than methods of the RMG class?